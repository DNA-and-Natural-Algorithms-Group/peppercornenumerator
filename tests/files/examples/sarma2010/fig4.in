# Figure 4 from Sarma2010report.pdf
# Correctly computes the steps shown, plus a bunch more subsequent steps and cross-interaction steps, in detailed mode.
# But hangs when trying to compute the condensed reactions.

domain 1 : long
domain 2 : long
domain 3 : short
domain 4 : long
domain 5 : short
domain 6 : long
domain 7 : short

strand h1 : 1
strand h2 : 1*
strand a1 : 3* 2* 1* 5 6
strand a2 : 1 2 3 4
strand b1 : 6 7
strand b2 : 7* 6*  5* 1 2 3

complex top :
h1
.

complex bot :
h2
.

complex com1 :
a1 a2
(((.. + ))).

complex com2 :
b1 b2
(( + ))....



