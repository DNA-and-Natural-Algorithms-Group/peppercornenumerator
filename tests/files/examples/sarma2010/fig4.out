###### Enumerated Output ######


# Domains 
sequence 1 = : 12
sequence 2 = : 12
sequence 3 = : 6
sequence 4 = : 12
sequence 5 = : 6
sequence 6 = : 12
sequence 7 = : 6


# End-state Complexes 
com1
3* 2* 1* 5 6 + 1 2 3 4
(((..+))).

22
3* 2* 1* 5 6 + 7* 6* 5* 1 2 3
(((((+.)))))

12
1 2 3 4
....

37
1 2 3 4 + 1*
(...+)

15
6 7
..

com2
6 7 + 7* 6* 5* 1 2 3
((+))....

36
6 7 + 7* 6* 5* 1 2 3 + 1*
((+)).(..+)

top
1
.

35
1 + 1*
(+)

bot
1*
.

###############################


# Fast (Transition) Complexes 
13
3* 2* 1* 5 6 + 6 7 + 7* 6* 5* 1 2 3
((((.+((+))))))

20
3* 2* 1* 5 6 + 6 7 + 7* 6* 5* 1 2 3
(((((+.(+))))))

5
3* 2* 1* 5 6 + 6 7 + 7* 6* 5* 1 2 3 + 1 2 3 4
((((.+((+)))...+))).

8
3* 2* 1* 5 6 + 6 7 + 7* 6* 5* 1 2 3 + 1 2 3 4
(((((+.(+)))...+))).

45
3* 2* 1* 5 6 + 6 7 + 7* 6* 5* 1 2 3 + 1* + 1 2 3 4
((((.+((+))))..+(+))).

46
3* 2* 1* 5 6 + 6 7 + 7* 6* 5* 1 2 3 + 1* + 1 2 3 4
(((((+.(+))))..+(+))).

41
3* 2* 1* 5 6 + 6 7 + 7* 6* 5* 1 2 3 + 1* + 1 2 3 4
((((.+((+)))(..+)+))).

44
3* 2* 1* 5 6 + 6 7 + 7* 6* 5* 1 2 3 + 1* + 1 2 3 4
(((((+.(+)))(..+)+))).

14
3* 2* 1* 5 6 + 7* 6* 5* 1 2 3 + 1 2 3 4
(((((+.))...+))).

52
3* 2* 1* 5 6 + 7* 6* 5* 1 2 3 + 1* + 1 2 3 4
(((((+.)))..+(+))).

64
3* 2* 1* 5 6 + 7* 6* 5* 1 2 3 + 1* + 1 2 3 4
(((((+.))(..+)+))).

###############################


# Reactions 
com1 + com2 -> 5 
com1 + 36 -> 41 
22 + 15 -> 20 
12 + bot -> 37 
com2 + bot -> 36 
top + bot -> 35 
13 -> 20 
20 -> 13 
5 -> 13 + 12 
5 -> 8 
8 -> 20 + 12 
8 -> 5 
45 -> 13 + 37 
45 -> 46 
46 -> 20 + 37 
46 -> 45 
41 -> 44 
44 -> 41 
14 -> 22 + 12 
52 -> 22 + 37 
45 -> 41 
46 -> 44 
41 -> 45 
44 -> 46 
52 -> 64 
64 -> 52 
20 -> 22 + 15 
5 -> com1 + com2 
8 -> 14 + 15 
46 -> 52 + 15 
41 -> com1 + 36 
44 -> 64 + 15 
